buildpack.md

```diff
@@ -162,12 +166,38 @@ Executable: `/bin/build <layers[EIC]> <platform[AR]> <plan[ER]>`, Working Dir: `
 
 Using the [Layer Content Metadata](#layer-content-metadata-toml) provided by a buildpack in a `<layers>/<layer>.toml` file, the lifecycle MUST determine:
 
-- Whether the layer directory in `<layers>/<layer>/` should be available to the app (via the `launch` boolean).
-- Whether the layer directory in `<layers>/<layer>/` should be available to subsequent buildpacks (via the `build` boolean).
-- Whether and how the layer directory in `<layers>/<layer>/` should be persisted to subsequent builds of the same OCI image (via the `cache` boolean).
+- Whether the layer directory `<layers>/<layer>/` should be available to the app (via the `launch` boolean).
+- Whether the layer directory `<layers>/<layer>/` should be available to subsequent buildpacks (via the `build` boolean).
+- Whether the layer directory `<layers>/<layer>/` should be persisted and made available to subsequent builds of the same OCI image (via the `cache` boolean).
 
 All combinations of `launch`, `build`, and `cache` booleans are valid. When a layer declares more than one type (e.g. `launch = true` and `cache = true`), the requirements of each type apply.
 
+The following table illustrates the behavior depending on the value of each flag.
+Note that the lifecycle only restores layers from the cache due to performance reasons.
```

Comment:
Is it correct to write the following:
`Note that the lifecycle only restores layers from the cache, never from the previous image, due to performance reasons.`
?

Reply:
Left the "performance" part outside, since we don't use to write such things in the spec.
Please see https://github.com/buildpacks/spec/pull/209/commits/3cd0f8be71cf90f563cbaa788ad1758f7d00f3ad.